hieradata/common/roles/api.yaml

```diff
@@ -27,14 +27,15 @@ profile::base::network::manage_dummy:                               true
 profile::highavailability::loadbalancing::haproxy::manage_haproxy:  true
 profile::highavailability::loadbalancing::haproxy::manage_firewall: true
 profile::highavailability::loadbalancing::haproxy::firewall_ports:
-  public:    ['5000', '8774', '8776', '9292', '9696', '80', '443']
-  internal:  ['5000', '8774', '8776', '9292', '9696', '35357']
+  public:    ['5000', '8774', '8776', '9292', '9696', '80', '443', '6082']
+  internal:  ['5000', '8774', '8776', '9292', '9696', '35357', '6082']
```

Comment:
Det er ingen frontend for internal, alt kommer på public så vi trenger ikke åpne brannmur for internal.

Reply:
hvis jeg fjerne portnummeret fra internal, så stopper brannmuren forbindelsen i api.

Reply:
Hmm. Husker det var problemer med brannmuren, men er det noe som lytter på <ipaddress_trp1>:6082 på api?